content/docs/operating/configuration.md

```diff
@@ -83,6 +83,20 @@ dynamically discovered using one of the supported service-discovery mechanisms.
 Additionally, `relabel_configs` allow advanced modifications to any
 target and its labels before scraping.
 
+If the targets require authentication, the following options are available:
+
+* `basic_auth` - sets the `Authorization` header on every scrape request with the
+configured username & password.
+* `client_cert` - configures the scrape request to use
+[mutual TLS](https://en.wikipedia.org/wiki/Mutual_authentication) with the
+configured certificate & key.
+* `bearer_token` - sets the `Authorization` header on every scrape request with
+the configured bearer token.
+* `bearer_token_file` - sets the `Authorization` header on every scrape request
+with the bearer token read from the configured file.
+
+See below for example use.
```

Comment:
It's not really an example use, but more of a reference of where those options go, right? How about "See below for the configuration of these authentication options.".